launch/launch/actions/declare_launch_argument.py

```diff
@@ -90,7 +93,10 @@ def __init__(
         else:
             self.__default_value = normalize_to_list_of_substitutions(default_value)
         self.__description = description
+        if arg_choices is not None:
+            self.__description += " Valid choices are: " + str(arg_choices)
```

Comment:
@v-lopez +1 to list valid choices in the description, but `self.__description` may actually be `'no description given'`. Perhaps `description` should be an `Optional` and be better handled here?

Reply:
Ok, which message would you expect if no `description` is provided, but `choices` is?

Reply:
The one you proposed looks fine.